o.n.core/src/org/netbeans/core/ui/ProductInformationPanel.java

```diff
@@ -44,7 +44,9 @@
 import javax.swing.event.HyperlinkEvent;
 import javax.swing.event.HyperlinkListener;
 import org.netbeans.core.actions.HTMLViewAction;
+
 import static org.netbeans.core.ui.Bundle.*;
```

Comment:
Move the static import up to a separate paragraph above the classic imports.

Reply:
Is there a setting for that in NetBeans?